README.md

```diff
@@ -168,6 +168,36 @@ Simply run command with appending [task variable](https://taskfile.dev/usage/#va
 sh taskw aem:build AEM_BUILD_ARGS="-PfedDev -DskipTests -pl '!ui.tests'"
 ```
 
+## SSL by Default support
```

Comment:
this is wrong place to put that docs; I am aware that there is no dedicated place for `Examples` right now so how about introducing such section 

below "Configuration" and above "Contributing"

<img width="763" alt="image" src="https://github.com/wttech/aemc/assets/81212505/ec0331bd-2662-406e-8318-e6ef24113a3d">

```
* Examples
  *  SSL by Default
```

please remember to update TOC as well